src/tree/registries/azure/AzureAccountTreeItem.ts

```diff
@@ -28,4 +30,25 @@ export class AzureAccountTreeItem extends AzureAccountTreeItemBase implements IR
     public createSubscriptionTreeItem(subContext: ISubscriptionContext): SubscriptionTreeItem {
         return new SubscriptionTreeItem(this, subContext);
     }
+
+    public async loadMoreChildrenImpl(_clearCache: boolean, context: IActionContext): Promise<AzExtTreeItem[]> {
+        const treeItems: AzExtTreeItem[] = await super.loadMoreChildrenImpl(_clearCache, context);
+
+        if (treeItems.length === 1 && treeItems[0].commandId === 'extension.open') {
```

Comment:
So this command causes the extension to be open in the gallery, but not necessarily to be installed. Should we have a time-out for the polling loop below, in case the user decides they don't want to install the extension after all?

Reply:
Added 10 min timeout.